app/views/admin/users/new.html.erb

```diff
@@ -0,0 +1,36 @@
+<% content_for :page_title do %>
+  <div class='col-md-offset-5 col-xs-offset-5 page-title'>
+    <%= link_to t(:user_page_title, scope: :action), admin_users_url%> / 
+    <%= t(:new, scope: [:action]) %>
+  </div>
+<% end %>
+
+<div class="form-div col-md-6 col-md-offset-3 col-xs-8 col-xs-offset-2">
+  <%= form_with(scope: @user, url: admin_users_path, local: true) do |form| %>
+    <fieldset>
+      <div class="form-group">
+        <h4 class= 'col-md-offset-4'>Create Customer Account</h4>
+      </div>
+
+      <%= render 'form', form: form, user: @user, email_disable_status: false %>
+      
+      <div class="form-group <%= 'has-error' if has_error?(:password, @user) %>">
+        <%= form.label :password, class: (@user.new_record? ? 'required-field' : '')  %>
+        
+        <%= form.password_field :password, id: :user_password, class: 'form-control', placeholder: '******' %>
+        <% if @user.persisted? %>
```

Comment:
see if this can be move into form partial now
as admin will be able to send user password now